cf-macro/template.yaml

```diff
@@ -0,0 +1,31 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: cf-macro-app
+    Description: Creates a Macro that can auto-generate Cloudwatch Alarms and Dashboard.
```

Comment:
"Creates the *SLIC Watch* CloudFormation macro to enable a Transform to generate Cloud*W*atch Alarms and a Dashboard."